src/test/java/com/kodilla/ecommercee/repository/ProductRepositoryTestSuite.java

```diff
@@ -0,0 +1,128 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Group;
+import com.kodilla.ecommercee.domain.Product;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+public class ProductRepositoryTestSuite {
+    @Autowired
+    private ProductRepository productRepository;
+    @Autowired
+    private GroupRepository groupRepository;
+
+    @Test
+    public void testSavingNewProductObject() {
+        //Given
+        Group group = groupRepository.save(new Group(10L, "test group_1"));
```

Comment:
Przy tworzeniu nowych obiektów unikałbym raczej podawania konkretnego `id` - jeśli zdarzy się, że obiekt z takim `id` będzie istniał w bazie danych, to zamiast utworzyć nowy obiekt, nadpiszesz ten istniejący. Najlepiej tworząc nowy obiekt pozostawić `id` nieustawione (w przypadku pełnego konstruktora można `id` ustawić na `null`).